docs/prover/cli/options.md

```diff
@@ -647,11 +647,7 @@ Options that help reduce the running time
 
 ### `--method`
 
-**What does it do?**
-Parametric rules will only verify the method with the given signature, instead of all public and external methods of the contract. Note that you will need to wrap the method's signature with quotes, as the shell doesn't interpret parenthesis correctly otherwise.
-
-**When to use it?**
-When you are trying to solve/understand a counterexample of a parametric rule on a specific method.
+See {ref}`--method`
```

Comment:
The only issue imo is that both here and in the [--method](https://docs.certora.com/en/latest/docs/prover/cli/options.html#method) option, the example we provide is string and not a list.

Reply:
added an example of command-line multiple methods. For the conf case it;s just like all the other flags we have that support lists, I don;t think we need any special docs for it